jf_agent/jf_jira/jira_download.py

```diff
@@ -33,9 +33,9 @@ def download_users(jira_connection, gdpr_active, quiet=False):
     jira_users = _search_all_users(jira_connection, gdpr_active)
 
     # Some jira instances won't return more than one page of
-    # results.  If we have seen exactly 1000 results, try
+    # results.  If we have seen approximately 1000 results, try
     # searching a different way
-    if len(jira_users) == 1000:
+    if 950 < len(jira_users) < 1000:
         jira_users = _users_by_letter(jira_connection, gdpr_active)
```

Comment:
nit: I might add a log file message so we can see if we're hitting this codepath

Reply:
+1 to this

Reply:
👍